ivy/functional/frontends/numpy/statistics/order_statistics.py

```diff
@@ -35,6 +43,41 @@ def _cpercentile(N, percent, key=lambda x: x):
     return d0 + d1
 
 
+@handle_numpy_out
+@handle_numpy_dtype
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+
+def percentile(
+    a,
+    q,
+    axis=None,
+    interpolation='linear',
+    keepdims=False
+):
+    a = ivy.array(a)
+    q = ivy.divide(q, 100.0)
+    q = ivy.array(q)
+
+    if not ivy.all(ivy.logical_and(q >= 0, q <= 1)):
+        ivy.logging.warning("Percentiles must be in the range [0, 100]")
+        return []
+
+    if axis is None:
+        nanlessarray = ivy.where(ivy.isnan(a), ivy.inf, a)
+        nanlessarray = ivy.reshape(nanlessarray, (-1,))
+        resultarray = ivy.percentile(nanlessarray, q)
+        return resultarray
+    elif axis == 1:
+        nanlessarrayofarrays = ivy.where(ivy.isnan(a), ivy.inf, a)
+        resultarray = ivy.percentile(nanlessarrayofarrays, q, axis=1, keepdims=keepdims)
+        return resultarray
+    elif axis == 0:
+        nanlessarrayofarrays = ivy.where(ivy.isnan(a), ivy.inf, a)
+        nanlessarrayofarrays = ivy.swapaxes(nanlessarrayofarrays, 0, 1)
+        resultarray = ivy.percentile(nanlessarrayofarrays, q, axis=0, keepdims=keepdims)

```

Comment:
It seems `ivy.percentile` does not exist

Reply:
I have made some changes